docs/serverless-v2/08-03-bind-service-instance.md

```diff
@@ -0,0 +1,148 @@
+      repositories:
+      - url: https://github.com/kyma-project/addons/releases/download/0.11.0/index-testing.yaml
+    EOF
+    ```
+
+3. Check if the Addon CR was created successfully. The CR phase should state `Ready`:
+
+    ```bash
+    kubectl get addons $NAME -n $NAMESPACE -o=jsonpath="{.status.phase}"
+    ```
+
+4. Create a ServiceInstance CR. You will use the provisioned [Redis](https://redis.io/) service with its `micro` plan:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: servicecatalog.k8s.io/v1beta1
+    kind: ServiceInstance
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      serviceClassExternalName: redis
+      servicePlanExternalName: micro
+      parameters:
+        imagePullPolicy: Always
+    EOF    
+    ```
+
+5. Check if the ServiceInstance CR was created successfully. The last condition in the CR status should state `Ready True`:
+
+    ```bash
+    kubectl get serviceinstance $NAME -n $NAMESPACE -o=jsonpath="{range .status.conditions[*]}{.type}{'\t'}{.status}{'\n'}{end}"
+    ```
+
+6. Create a ServiceBinding CR that points to the newly created Service Instance in the **spec.instanceRef** field:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: servicecatalog.k8s.io/v1beta1
+    kind: ServiceBinding
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      instanceRef:
+        name: $NAME
+    EOF    
+    ```
+
+    > **NOTE:** If you use an existing Service Instance, change **spec.instanceRef.name** to the name of your Service Instance.
+
+7. Check if the ServiceBinding CR was created successfully. The last condition in the CR status should state `Ready True`:
+
+    ```bash
+    kubectl get servicebinding $NAME -n $NAMESPACE -o=jsonpath="{range .status.conditions[*]}{.type}{'\t'}{.status}{'\n'}{end}"
+    ```
+
+8. Create a ServiceBindingUsage CR:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: servicecatalog.kyma-project.io/v1alpha1
+    kind: ServiceBindingUsage
+    metadata:
+      name: $NAME
+      namespace: $NAMESPACE
+    spec:
+      serviceBindingRef:
+        name: $NAME
+      usedBy:
+        kind: knative-service
+        name: $NAME
+      parameters:
+        envPrefix:
+          name: "REDIS_"
+    EOF    
+    ```
+
+    - The **spec.serviceBindingRef** and **spec.usedBy** fields are required. **spec.serviceBindingRef** points to the Service Binding you have just created and **spec.usedBy** points to the lambda. More specifically, **spec.usedBy** refers to the name of the related KService CR (`name: $NAME`) and the cluster-specific [UsageKind CR](https://kyma-project.io/docs/components/service-catalog/#custom-resource-usage-kind) (`kind: knative-service`) that defines how Secrets should be injected to your lambda through the Service Binding.
+
+    > **NOTE:** Read [this](/components/service-catalog/#details-provisioning-and-binding) document to learn more about binding in Kyma.
```

Comment:
Actually, we already have a similar note and the link at the beginning of the doc, so let's remove this one as it is a duplicate.